umm-lib/test/cmr/umm/test/echo10/granule.clj

```diff
@@ -8,14 +8,15 @@
 
             [clojure.test.check.properties :refer [for-all]]
             [clojure.test.check.generators :as gen]
-            [clojure.string :as s]
-            [cmr.umm.test.generators.granule :as gran-gen]
+            [clojure.string :as string]
             [cmr.common.date-time-parser :as p]
+            [cmr.common.xml :as cx]
             [cmr.umm.echo10.granule :as g]
             [cmr.umm.echo10.echo10-core :as echo10]
             [cmr.umm.umm-collection :as umm-c]
             [cmr.umm.umm-granule :as umm-g]
-            [cmr.umm.test.echo10.echo10-collection-tests :as tc]))
+            [cmr.umm.test.echo10.echo10-collection-tests :as tc]
```

Comment:
nit: the imports are not sorted correctly and indentation is wrong.

Reply:
fixed.